Buildings/Controls/OBC/FDE/DOAS/EnergyWheel.mo

```diff
@@ -0,0 +1,276 @@
+within Buildings.Controls.OBC.FDE.DOAS;
+block EnergyWheel "This block commands the energy recovery wheel and associated bypass dampers."
+  parameter Real recSet(
+  final unit = "K",
+  final displayUnit = "degC",
+  final quantity = "ThermodynamicTemperature") = 7
+  "Energy recovery temerature difference set point.";
+
+   parameter Real Thys(
+  final unit = "K",
+  final displayUnit = "degC",
+  final quantity = "ThermodynamicTemperature") = 0.5
+  "Temperature Difference for hysteresis.";
+
+  parameter Real recSetDelay(
```

Comment:
timDelEneRec. Delay time period after temperature difference threshold is crossed for enabling energy recovery mode